### DISCUSSION TITLE
音乐插件无法显示的问题

### DISCUSSION THREAD

**Discussion by OscarZs** [tangly1024/NotionNext#710] [Q&A]
在本人网站上无法显示音乐插件。两天前成功显示过，后来就显示不出来了。

![image](https://user-images.githubusercontent.com/88417660/213871993-8a8d996a-7dac-4d34-94c3-f196b24c4213.png)

下面是我的代码设置
![image](https://user-images.githubusercontent.com/88417660/213871624-3297c777-04d2-4dd6-bde3-44527b8865f5.png)

下面是报错，我大概知道是一个CORS的问题，不会修orz

![image](https://user-images.githubusercontent.com/88417660/213871931-defa52c1-e2af-4b23-9238-8a7332aba9b1.png)

这个id的歌单是可以正常播放的

![1674310556097](https://user-images.githubusercontent.com/88417660/213870968-88c1eb78-04bf-4d13-9993-9fdc788a57e9.jpg)

**Comment by lifeafter619** [comment]
我也遇到了相同的问题

**Reply by OscarZs** [reply]
我最后一个一个导入了orz

**Comment by lifeafter619** [comment]
我在aplayer的issue和交流群中也发现了这个问题，貌似是他们的接口限制了跨域，好像也没有换自己接口的配置，暂时无解😂😮‍💨![image](https://user-images.githubusercontent.com/65111206/216220374-7e814c7e-7477-4805-8294-61eb37b767de.jpeg)

**Reply by OscarZs** [reply]
谢谢啦。
我最后没用歌单，把十几首单曲手动弄进去了😂

**Reply by lifeafter619** [reply]
> 谢谢啦。
> 我最后没用歌单，把十几首单曲手动弄进去了😂

好的👌，但是我的歌单有上百首，这也许不是最佳选择😂，不过我会继续跟进该问题，在有更好的解决方案时，我将pr进来🙏
